### ISSUE TITLE
big sur has read-only filesystem mounted at /

### ISSUE THREAD

**Issue by tesch1** [OpenVnmrJ/OpenVnmrJ#509]
**Describe the bug**

At least from macos Big Sur onward, the root fs is strongly protected - this causes the creation of a symbolic link from /vnmr/ -> .OpenVnmrJ..app/.../vnmr to fail during installation.

**To Reproduce**
- Install OpenVnmrJ.pkg on osx > 11.2.x
- notice that /vmnr/ link is missing

**Workarounds**

There are some fancy things that are bad for the security of your system, but appear to be a possible work around for this issue: https://developer.apple.com/forums/thread/649832 .  In short: add the following line to a file /etc/synthetic.conf and reboot.

```
% sudo sh -c 'echo "vnmr\t/Applications/OpenVnmrJ_2.1.app/Contents/Resources/OpenVnmrJ" >> /etc/synthetic.conf'
% cat /etc/synthetic.conf
vnmr	/Applications/OpenVnmrJ_2.1.app/Contents/Resources/OpenVnmrJ
% reboot
...
% ls /vnmr/
```

**Another fix**
The installation dir of vnmrj could be made directory-independent, perhaps pulling it from an environment variable.

**Comment by DanIverson** [comment]
Version 2.1 of OVJ for Mac has the synthetic.conf fix. see src/macos/postinstall. I see the problem. Apple changed the numbering scheme for Big Sur. Catalina is 10.15.x  Big Sur 11.0.x I check that the second number is >= 15. Need to change to either first number > 10 or second number >= 15.

**Comment by DanIverson** [comment]
This should be fixed with PR #510
